lib/compute/operation.js

```diff
@@ -160,6 +166,7 @@ function Operation(scope, name) {
   this.completeListeners = 0;
   this.hasActiveListeners = false;
   this.name = name;
+  this.progress = null;
```

Comment:
Time for one of these:

![](https://camo.githubusercontent.com/c84dacb67e3d84047c2d18c10a1b86228d258a97/68747470733a2f2f33312e6d656469612e74756d626c722e636f6d2f35356431373063383938623234303738336463333332313234626264313937642f74756d626c725f696e6c696e655f6e34666f616672613043317365773830682e6a7067)

`null` is typically used to reference a placeholder for an object that doesn't exist yet. In this case, it's only going to be a number, so the "correct" substitute is simply `undefined`. Do you hate me?